### PR TITLE
Fixes #17485: Add a banner on the docs of non-released/deprecated versions

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://issues.rudder.io/issues/17485

### CODE REVIEW

File: rudder-theme/src/partials/toolbar.hbs [hunk 1]
```diff
@@ -1,3 +1,9 @@
+{{~#if (eq page.componentVersion "6.1") }}
```
Comment (amousset) [inline rudder-theme/src/partials/toolbar.hbs:1]:
```
 {{~#if (eq page.componentVersion.version "6.1") }}                                                                                                            
```

**Comment by RaphaelGauthier** [comment]
PR rebased

**Comment by RaphaelGauthier** [comment]
Commit modified

**Comment by RaphaelGauthier** [comment]
Commit modified

**Pull request merged**